accounts-db/src/accounts_db.rs

```diff
@@ -2714,9 +2714,9 @@ impl AccountsDb {
                 return;
             }
             if let Some(storage) = self.storage.get_slot_storage_entry(slot) {
-                storage.accounts.scan_accounts_stored_meta(|account| {
-                    let pk = account.pubkey();
-                    match pubkey_refcount.entry(*pk) {
+                storage
+                    .accounts
+                    .scan_pubkeys(|pubkey| match pubkey_refcount.entry(*pubkey) {
```

Comment:
1. this fn is a very slow debugging method.
2. this maybe should go in a new pr because it moves to a new api (`scan_pubkeys`)